python/cli/prelude_cli/views/iam.py

```diff
@@ -18,12 +18,16 @@ def iam(ctx):
 @iam.command('create-account')
 @click.pass_obj
 @handle_api_error
+@click.option('-m', '--method',
+              help='provide a registration method',
+              default='email', show_default=True,
+              type=click.Choice(['email', 'bypass'], case_sensitive=False))
```

Comment:
providing options seems unnecessary. couldn't we just make the click.option a flag that defaults bypass=False?

doing that, I'd avoid sending the 'method' in the body but instead use it as a query parameter, so you could do something like ``POST:/iam/account?bypass=1``

Reply:
Unless you're tying this in somehow in the OAUTH version... I think either way though, I'd opt for the method being in the query params vs the body, as it's a property to direct the API toward a specific path... so ``/iam/account?method=bypass`` could be an alternative option.

Reply:
updated to the latter as a query param